### DISCUSSION TITLE
Create URL to open Organic Maps on Android using known coordinates

### DISCUSSION THREAD

**Discussion by EvgenyZaretskiy** [organicmaps/organicmaps#9153] [Q&A]
Is there a way to create a link to open Organic Maps by one click using coordinates? Like https://osmand.net/map?pin=12.7775097,99.9707717#22 or I should definitely encode coordinates to have format https://omaps.app/0yzF7NW2HN/ABC and it is the only format to open specific point on the map when Organic Maps started by click?

**Comment by biodranik** [comment]
Check https://omaps.app/api

**Reply by EvgenyZaretskiy** [reply]
OK, No way to specify coordinates in URL for Organic Maps. BTW, I can't dig so deep in a the source code, but is libge0 precompiled library for linux available to use it for encodind?

**Reply by biodranik** [reply]
Did you miss an example on the link above? https://omaps.app/map?v=1&ll=22.17319,-159.65687&n=Kalalau%20Camping&ll=22.17168,-159.66096&n=Dream%20Beach&ll=22.17182,-159.65776

**Reply by EvgenyZaretskiy** [reply]
OMG, I really did. Thank a lot, that's working. Do you know, if Organic Maps coming from the same source wit Maps.Me, in Map's me do they have something similar to open the link ge0.me using known coordinates?
